mycity/mycity/test/unit_tests/test_arcgis_utils.py

```diff
@@ -0,0 +1,27 @@
+import mycity.test.unit_tests.base as base
+import unittest.mock as mock
+import mycity.test.test_constants as test_constants
+import mycity.utilities.arcgis_utils as arcgis_utils
+import mycity.mycity_controller as my_con
+import mycity.intents.intent_constants as intent_constants
+
+
+class ArcgisUtilitiesTestCase(base.BaseTestCase):
+    
```

Comment:
We should refactor to these. We had some problems with the `crime_incidents_intent` calling out to its service even though we monkey-patched the intent

Reply:
On that note, it would be good to have an integration test for this intent so that we can detect any changes in how the response from the service is formatted

Reply:
@wesdrew what am I refactoring?

Reply:
I would explicitly pass in any object that will would initiate a call that goes out to the Internet. Or, rather you could set a static variable within your intent that should be called in lieu of that function/object that goes out on the wire when that variable is set

Reply:
We can talk next week. If @jmartini or @curranjm are fine with picking that refactor up in another pull request, that's OK with me, too

Reply:
@wesdrew are you referring to mocking requests?

Reply:
@wesdrew This test isn't making any calls to the internet other than the initial request of each function, which I've mocked.